Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -1818,8 +1816,7 @@ namespace ContinuousLinearMap
 variable {E' : Type*} [NormedAddCommGroup E'] [InnerProductSpace 𝕜 E']
 
 -- Note: odd and expensive build behavior is explicitly turned off using `noncomputable`
-set_option maxHeartbeats 500000 in
-set_option synthInstance.maxHeartbeats 100000 in
+set_option synthInstance.maxHeartbeats 40000 in
```

Comment:
Is the sentiment of [this thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/let's.20not.20crank.20maxHeartbeats.20as.20low.20as.20possible) relevant here?

Reply:
What sentiment do you mean?

Reply:
Just that we should leave some headroom on `maxHeartbeats` to avoid annoying brittleness to subsequent changes.

Reply:
That's why I didn't try to find the precise bound.